test/utils/interop-daemon-spawner/go.js

```diff
@@ -40,8 +40,7 @@ class GoDaemon {
         this.node = node
         this.node.setConfig('Bootstrap', '[]', cb)
       },
-      (res, cb) => this.node.startDaemon(cb),
-      // (res, cb) => this.node.startDaemon(this.flags, cb),
```

Comment:
It is this change causing the latest error

Reply:
The 2nd line should not be commented out.  It's needed to pass `--enable-pubsub-experiment` to the daemon.